src/uu/du/src/du.rs

```diff
@@ -340,8 +341,12 @@ fn du(
                             let description = format!("cannot access {}", entry.path().quote());
                             let error_message = "Permission denied";
                             show_error_custom_description!(description, "{}", error_message);
+                            set_exit_code(1);
```

Comment:
I think this can be cleaned up some more with `UResult`, which will print the right error message automatically when converting from an `io::Error`. However, that is a more involved refactor, so I'll approve and leave it up to you whether you want to do this.

Reply:
OK, I will land this and open an new issue and maybe work on it